src/js/Content/Features/Store/Common/FExtraLinks.js

```diff
@@ -38,128 +45,127 @@ export default class FExtraLinks extends Feature {
 
     apply() {
 
+        const isAppPage = this._type === "app";
+        const appName = typeof this.context.appName === "string" && HTMLParser.clearSpecialSymbols(this.context.appName);
+
         // Note: Links should be rendered in the same order as displayed on the options page
         const links = [
             {
-                "id": "showitadlinks",
-                "className": "itad_ico",
+                "link": `https://steamcommunity.com/app/${this._gameid}/`,
+                "text": "Community Hub",
```

Comment:
Isn't that a localizable string?

Reply:
I think we have "Community", but not "Community Hub", will need a new one